src/Components/dashboard/EditEvents.js

```diff
@@ -1,5 +1,203 @@
+import { useState, useEffect } from "react";
+import { useLocation, Navigate } from "react-router-dom";
+
+import { TextField } from "@mui/material";
+import DeleteIcon from "@mui/icons-material/Delete";
+import DoneIcon from "@mui/icons-material/Done";
+
+import dayjs from "dayjs";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import { MobileDatePicker } from "@mui/x-date-pickers/MobileDatePicker";
+
+import AvatarUser from "../avatarUser/AvatarUser";
+import {
+  styles,
+  getEventsFromLocalStorage,
+  saveEventsInLocalStorage,
+} from "../../utils";
+import "./styleEditEvents.scss";
+
 const EditEvent = () => {
-  return <div>Editar evento</div>;
+  const location = useLocation();
+  const eventToEdit = location.state.eventToEdit;
+  const eventsInLocalStorage = getEventsFromLocalStorage();
+  let events = eventsInLocalStorage.events || [];
+  const indexEventToEdit = events.findIndex(
+    (event) => event.id === eventToEdit
+  );
+  const event = events[indexEventToEdit] || events[0];
+  const [dateEvent, setDateEvent] = useState(dayjs(event.date));
+  const [timeEvent, setTimeEvent] = useState(event.time);
+  const [titleEvent, setTitleEvent] = useState(event.nameEvent);
+  const [descriptionEvent, setDescriptionEvent] = useState(
+    event.descriptionEvent
+  );
+  const [capacityPeopleEvent, setCapacityPeopleEvent] = useState(
+    event.capacity
+  );
+  const [itemToDraw, setItemToDraw] = useState("");
+  const [backToDashboard, setBackToDashboard] = useState(false);
+  const dateFormats = {
+    time: "h:mm A",
+    customDate: "ddd MMM DD YYYY",
+  };
+  const { classes } = styles();
+  dayjs.locale("en");
+
+  useEffect(() => {
+    setItemToDraw(drawEventToEdit);
```

Comment:
call a helper function  called `getEventData` in here to simulate a call to the server, please check the function we created `getAvatarAndName`